src/main/java/org/gridsuite/actions/server/DataSourceConfig.java

```diff
@@ -26,12 +26,13 @@
     public DataSource getDataSource(Environment env) {
         DataSourceBuilder dataSourceBuilder = DataSourceBuilder.create();
         String driver = env.getRequiredProperty("driver");
-
-        dataSourceBuilder.driverClassName(env.getRequiredProperty("driverClassName"));
         if (driver.equals("h2")) {
             dataSourceBuilder.url(env.getRequiredProperty("url"));
         } else if (driver.equals("postgresql")) {
-            dataSourceBuilder.url(env.getRequiredProperty("url") + "/" + env.getRequiredProperty("database"));
+            dataSourceBuilder.url("jdbc:postgresql://" +
+                    env.getRequiredProperty("host") + ":" +
+                    env.getRequiredProperty("port") + "/" +
+                    env.getRequiredProperty("database"));
```

Comment:
database value shouldn't be in the database.properties because this file will be shared by multiple webservices (in deployment)
create a  class attribute to hold the value :     private static final String DATABASE_NAME="actions";

Reply:
@niconoir  i think that this remark concern the directory server too

Reply:
It is not the job of the env.getRequiredProperty method to look a property in the multiple property sources defined by the @PropertySource annotations, and then retrieve the 'database' property defined in the project database.properties file, because it is not defined in the database.properties file in deployment ?